src/routes/Forms.jsx

```diff
@@ -42,18 +42,37 @@ const Forms = () => {
         justifyContent: 'center'
       }}
     >
-      <Button
-        variant="contained"
-        color="secondary"
-        size="small"
-        aria-label="Clear Client Button"
-        startIcon={<Remove />}
-        onClick={() => setSelectedUser('')}
-        sx={{ margin: '2rem 0 1rem' }}
-      >
-        Clear Client
-      </Button>
-      <Typography>Client selected: {selectedUser.person || 'No client selected'}</Typography>
+      {selectedUser.person ? (
+        <>
+          <Button
+            variant="contained"
+            color="secondary"
+            size="small"
+            aria-label="Clear Client Button"
+            startIcon={<Remove />}
+            onClick={() => setSelectedUser('')}
+            sx={{ margin: '2rem 0 1rem' }}
+          >
+            Clear Client
+          </Button>
+          <Typography variant="body2">Client selected: {selectedUser.person}</Typography>
+        </>
+      ) : (
+        <>
```

Comment:
Already mentioned in earlier comment, don't think we need to disable this since we know where the problem is now.

Reply:
Duplicate